lib/pkg.ml

```diff
@@ -204,20 +193,22 @@ let homepage_is_on_github p =
   opam_homepage_sld p >>| function None -> false | Some sld -> sld = "github"
 
 let path_of_distrib p =
+  dev_repo_is_on_github p >>= fun repo_on_gh ->
+  homepage_is_on_github p >>= fun hp_on_gh ->
+  name p >>= fun name ->
+  tag p >>= fun tag ->
   let basename =
     match p.distrib_file with
     | Some f -> Fpath.basename f
-    | None -> "$(NAME)-$(TAG).tbz"
+    | None -> strf "%s-%a.tbz" name Vcs.Tag.pp tag
   in
-  dev_repo_is_on_github p >>= fun a ->
-  homepage_is_on_github p >>= fun b ->
-  let uri =
-    (if a || b then "releases/download/$(TAG)/" else "releases/") ^ basename
+
```

Comment:
I'm surprised ocamlformat didn't get rid of that empty line but either I think we should drop it! I don't think it's intentional anyway, is it?